pkg/server/controllers.go

```diff
@@ -418,6 +440,25 @@ func (s *Server) installWorkspaceScheduler(ctx context.Context, config *rest.Con
 		return err
 	}
 
+	s.AddPostStartHook(postStartHookName(clusterworkspacetype.ControllerName), func(hookContext genericapiserver.PostStartHookContext) error {
+		logger := klog.FromContext(ctx).WithValues("postStartHook", postStartHookName(clusterworkspacetype.ControllerName))
+		if err := s.waitForSync(hookContext.StopCh); err != nil {
+			logger.Error(err, "failed to finish post-start-hook")
+			return nil // don't klog.Fatal. This only happens when context is cancelled.
+		}
+		go workspaceTypeController.Start(ctx, 2)
+		return nil
+	})
+
+	// TODO(ncdc): this is here because the call to bootstrap.NewController below needs a bootstrap client for kcp,
+	// but it takes in a kcpclient.Interface, not kcpclient.ClusterInterface. The types on Server are all
+	// *.ClusterInterface. We'll be able to unify things once the work to simplify and consolidate our clients is
+	// done.
+	bootstrapConfig := rest.CopyConfig(config)
+	bootstrapConfig = rest.AddUserAgent(kcpclienthelper.SetMultiClusterRoundTripper(bootstrapConfig), bootstrap.ControllerNameBase)
```

Comment:
Since this is just for the "universal" CWT bootstrapper, the name and user-agent should reflect that (i.e. include "universal" somewhere)